articles/manual/putting-it-all-together.md

```diff
@@ -0,0 +1,178 @@
+---
+# Mandatory fields. See more on aka.ms/skyeye/meta.
+title: Putting it all together - Q# techniques | Microsoft Docs 
+description: Putting it all together - Q# techniques
+uid: microsoft.quantum.manual.puttingittogether
+author: QuantumWriter
+ms.author: Christopher.Granade@microsoft.com
+ms.date: 12/11/2017
+ms.topic: article
+# Use only one of the following. Use ms.service for services, ms.prod for on-prem. Remove the # before the relevant field.
+# For Quantum products none of these categories have been defined  yet.
+# ms.service: service-name-from-white-list
+# ms.prod: product-name-from-white-list
+# ms.technology: tech-name-from-white-list
+---
+
+# Putting It All Together: Teleportation #
```

Comment:
This article feels a bit out of date as well, in that it's heavily based on the circuits concept article, and uses somewhat different names for each quantum operations from what we eventually settled on in the Microsoft.Quantum.Intrinsic namespace.